_posts/2025-08-05-fcmp++-contest-final.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: FCMP++ Optimization Competition Results
+summary: Announcing the results of the FCMP++ optimization competition
+tags: [crypto, announcements]
+author: Justin Berman (j-berman) and jeffro256
+---
+
+We held [a contest](./2025-04-05-fcmp++-contest.md) to optimize two libraries used in Monero's proposed upgrade to [FCMP++]({{site.baseurl}}/2024/04/27/fcmps.html): [helioselene](https://github.com/kayabaNerve/fcmp-plus-plus/tree/78754718faa21f0a5751fbd30c9495d7f7f5c2b1/crypto/helioselene) and [ec-divisors](https://github.com/kayabaNerve/fcmp-plus-plus/tree/78754718faa21f0a5751fbd30c9495d7f7f5c2b1/crypto/divisors). We have officially declared the winners of the contest!
+
+## EC Divisors (250 XMR prize)
+
+Winner: [@fabrizio-m](https://github.com/fabrizio-m)
+
+Fabrizio sped up the divisors library by over 95%. You can see the excellent submission [here](https://github.com/fabrizio-m/fcmp-competition/pull/1).
+
+`ec-divisors` is a library implementing the calculation of divisors of elliptic curve points. These lead to efficient verification of scalar multiplications, [as posited by Liam Eagen](https://eprint.iacr.org/2022/596). FCMP++ uses EC Divisors to improve efficiency generally. This competition was focused on optimizing divisor construction, which affects transaction construction.
+
+Managing slow divisors construction was a pain point for the FCMP++ integration. Fabrizio's 95%+ improvement effectively renders divisors construction negligible. Thanks to Fabrizio, we can now significantly simplify the final FCMP++ integration, and improve the user experience.
+
+## helioselene (100 XMR 1st place prize, 30 XMR 2nd place prize)
+
+1st place: [@lederstrumpf](https://github.com/Lederstrumpf)
+2nd place: [@rafael-xmr](https://github.com/rafael-xmr/)
```

Comment:
```suggestion
1st place: [@lederstrumpf](https://github.com/Lederstrumpf)  
2nd place: [@rafael-xmr](https://github.com/rafael-xmr/)
```

I'm not 100% sure getmonero handles markdown newline like this, but adding 2 spaces to the end of the line usually works elsewhere

Reply:
Good spot, `<br>` works too without giving noisy git warnings